frontend/src/container/AppLayout/index.tsx

```diff
@@ -71,7 +72,14 @@ function AppLayout(props: AppLayoutProps): JSX.Element {
 	const isPremiumChatSupportEnabled =
 		useFeatureFlags(FeatureKeys.PREMIUM_SUPPORT)?.active || false;
 
+	const isChatSupportEnabled =
+		useFeatureFlags(FeatureKeys.CHAT_SUPPORT)?.active || false;
+
+	const isCloudUserVal = isCloudUser();
+
 	const showAddCreditCardModal =
```

Comment:
will this hide intercom for signoz.io users ?

Reply:
Yes

Reply:
This should be shown to only cloudUsers who don't have premium chat support enabled and who are in trail.